src/spell/modules/spell/spell/src/auth.rs

```diff
@@ -7,12 +7,14 @@ pub fn is_by_creator() -> bool {
     call_parameters.init_peer_id == call_parameters.service_creator_peer_id
 }
 
-/// returns true if call was made from the associated spell script
+/// returns true if call was made from the associated spell script.
+/// true if particle id has a form of `spell_<spell_id>_<counter>`
+/// and `is_by_creator` returns true.
```

Comment:
I meant, afaik we call `is_by_creator` here because the spell is run on behalf of the creator (aka worker_id), so if `is_by_creator` is false, that means it's not by spell but by some malicious source. If I'm correct in my deduction, we need to write it here to clarify it.

Reply:
oh, ok. We call it because anyone can set any particle id